### DISCUSSION TITLE
Hi from a statistician interested in AI policy

### DISCUSSION THREAD

**Discussion by stefan-stein** [OpenMined/courses#90] [Introduce Yourself]
Hi all,

I am a statistics PhD student and excited to be here. While I have a background in maths and coding, one of the reasons I am taking this course because I am interested in AI policy and I think data privacy should be an integral part of that. If you want to chat, feel free to reach out: https://www.linkedin.com/in/stefan-stein

Best wishes,

Stefan

**Comment by yemikifouly** [comment]
Hello Stefan! It is great to have you here! Welcome to the discussion board! 🙂 I am sure you will enjoy the concepts and discussions raised by the first course. 🙂 

Also, for you LinkedIn link, write out the full link starting with "https://". Otherwise, GitHub considers that it is a relative url and append it to the url of this page. 🙂

**Reply by stefan-stein** [reply]
Thanks for the warm welcome and the hint!
